pygfunction/properties.py

```diff
@@ -0,0 +1,122 @@
+        >>> mix = 'MEG'
+        >>> percent = 60
+        >>> fluid = gt.properties.Fluid(m_flow, T=T, P=P, mixer=mix, percent=percent)
+        >>> print(fluid.__repr__())
+
+        >>> # 5% methanol mixed with water water
+        >>> mix = 'MMA'
+        >>> percent = 5
+        >>> fluid = gt.properties.Fluid(m_flow, T=T, P=P, mixer=mix, percent=percent)
+        >>> print(fluid.__repr__())
+
+        >>> # ethanol / water
+        >>> mix = 'MEA'
+        >>> percent = 10
+        >>> fluid = gt.properties.Fluid(m_flow, T=T, P=P, mixer=mix, percent=percent)
+        >>> print(fluid.__repr__())
+    """
+    def __init__(self, m_flow, T: float = 293., P: float = 239220., mixer: str = 'MEG', percent: float = 0):
+        self.fluid_mix = 'INCOMP::' + mixer + '-' + str(percent) + '%'
+        if mixer == 'MEG':
+            pass
+        elif mixer == 'MPG':
+            pass
+        elif mixer == 'MMA':
+            pass
+        elif mixer == 'MEA':
+            pass
+        # add in else if's
+        else:
+            warnings.warn('It is unknown whether or not cool props has the mixing fluid requested, '
+                          'proceed with caution.')
+        self.m_flow = m_flow                            # Total fluid mass flow rate per borehole {kg/s}
+        self.T = T                                      # temperature of the fluid {K}
+        self.P = P                                      # pressure of the fluid in {Pa}
+        self.rho = self.density()                       # Density, {kg/m^3}
+        self.mu = self.dynamic_viscosity()              # Dynamic Viscosity, {Pa s} or {N s/ m^2}
+        self.nu = self.kinematic_viscosity()            # Kinematic Viscosity, {m^2/s}
+        self.cp = self.mass_heat_capacity()             # Mass specific Cp Specific Heat, {J/kg/K}
+        self.rhoCp = self.volumetric_heat_capacity()    # Volumetric heat capacity, {kJ/m3/K}
+        self.k = self.thermal_conductivity()            # Thermal conductivity, {W/m/K}
+        self.Pr = self.Prandlt_number()                 # Prandlt number
+
+    def __repr__(self):
+        return str(self.__class__) + '\n' + '\n'.join(
+            (str(item) + ' = ' + '{}'.format(self.__dict__[item]) for item in sorted(self.__dict__)))
+
+    def append_to_dict(self, dnary):
+        if len(list(dnary.keys())) == 0:
+            for item in sorted(self.__dict__):
+                dnary[item] = []
+        for item in sorted(self.__dict__):
+            dnary[item].append('{:.5E}'.format(self.__dict__[item]))
+
+    def density(self):
+        return PropsSI('D', 'T', self.T, 'P', self.P, self.fluid_mix)
+
+    def dynamic_viscosity(self):
+        return PropsSI('V', 'T', self.T, 'P', self.P, self.fluid_mix)
+
+    def kinematic_viscosity(self):
+        return self.mu / self.rho
+
+    def mass_heat_capacity(self):
+        return PropsSI('C', 'T', self.T, 'P', self.P, self.fluid_mix)
+
+    def volumetric_heat_capacity(self):
```

Comment:
See the other comment. Methods in the 'Pipe' and 'Network' classes use J/kg-degC.